cfpb/design-system
# Styling: Checkbox States (Error/Warning/Success)

closes #1883

## Changes
- Carried over the error/warning/success state styling from TextInput to Checkboxes
- Updated the Checkbox stories to show the new styling

## How to Test 
- Test Link: https://deploy-preview-1889--cfpb-design-system.netlify.app/
- **Code inspect** `form-fields.less` and ensure no unintended styling changes affect other components
- **Navigate to the `Checkboxes` story section and view the new styles

## Screenshot(s)

Added states

|Before|After|
|---|---|
|<img width="317" alt="Screenshot 2024-01-22 at 12 00 47 PM" src="https://github.com/cfpb/design-system/assets/13324863/724f604b-905c-4778-ae25-21f4956af650">|<img width="226" alt="Screenshot 2024-01-22 at 12 18 31 PM" src="https://github.com/cfpb/design-system/assets/13324863/f4f5ac65-ea31-40fd-9174-96fb1caf7bef">|
